internal/overlord/servstate/export_test.go

```diff
@@ -72,11 +72,13 @@ func FakeOkayWait(wait time.Duration) (restore func()) {
 	}
 }
 
-func FakeKillWait(kill, fail time.Duration) (restore func()) {
-	old1, old2 := killWait, failWait
-	killWait, failWait = kill, fail
+// FakeGraceKillWait changes both the defaultGracePeriod and killWaitDuration
+// respectively for testing purposes.
+func FakeGraceKillWait(gracePeriod, killWait time.Duration) (restore func()) {
+	old1, old2 := defaultGracePeriod, killWaitDuration
+	defaultGracePeriod, killWaitDuration = gracePeriod, killWait
 	return func() {
-		killWait, failWait = old1, old2
+		defaultGracePeriod, killWaitDuration = old1, old2
```

Comment:
Cool, thanks. One more thing. These (at the top of `servstate/manager_test.go`):

```go
	shortKillWait = 100 * time.Millisecond
	shortFailWait = 200 * time.Millisecond
```

Should probably be changed to `shortGracePeriod` and `shortKillWaitDuration`, and the latter could be changed to `100 * time.Millisecond` to match what it was before (now that it's additive).